bedrock/firefox/templates/firefox/mobile/index.html

```diff
@@ -39,12 +39,6 @@ <h3>{{ ftl('firefox-mobile-get-automatic-privacy-on-mobile') }}</h3>
           </div>
 
           <div class="mzp-c-hero-cta">
-            <div class="header-product-ctas hide-from-legacy-ie">
```

Comment:
There's still a button at the bottom of the page that might need removing? (clicking it currently does nothing)

![image](https://user-images.githubusercontent.com/400117/89894918-cef50280-dbd2-11ea-8b46-afef391ce28e.png)